task-microservice/src/main/resources/application.yml

```
@@ -45,6 +45,4 @@ springdoc:
   remove-broken-reference-definitions: true
   override-with-gateway-url: true
   server:
-    url: http://localhost:5000
-
-
+    url: http://localhost:5000
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)